### DISCUSSION TITLE
The ability to export/import bookmarks via the commandline

### DISCUSSION THREAD

**Discussion by R1yuu** [minbrowser/min#1767] [Ideas]
The ability to export and import Bookmarks in general already exists.
It would be amazing tho, if these features could get a commandline integration like:
min -importbookmarks <dir>
min -exportbookmarks <dir>

It would enable people to easily write script or such to sync bookmarks between devices.
In this context a command to delete specific/all bookmarks in general would be interesting aswell

**Comment by shalva97** [comment]
I also want something like this. At work Im forced to use popular browsers like Chrome or Firefox. 

exporting bookmarks via CLI would allow to automate sharing bookmarks across different browsers. This feature could also export tasks as if they where bookmarks.
